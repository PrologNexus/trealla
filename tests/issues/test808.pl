:- initialization(main).

main :-
	number_chars(N,"'\\\n4"),
	write(N), nl.
