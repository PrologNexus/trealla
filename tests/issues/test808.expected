   Warning: Initialization goal exception: error(syntax_error(unterminatedt),number_chars/2).
